### ISSUE TITLE
Title bar does not mesh well on the iPhone X

### ISSUE THREAD

**Issue by rmevans9** [BoostIO/boostnote-mobile#88]
It appears that you are using a older version of `native-base` that does not have the fix for the iPhone X header in. This makes the usability on iPhone X pretty terrible. 

I upgraded to the latest `native-base` (and `react-native` to support changes in `native-base`) and all seems mostly well. There are a few minor things I still need to try and work out like in the add note modal the "helper" buttons at the bottom are slightly off screen. Likely because of the differing header sizes.

**Comment by kazup01** [comment]
Hi @rmevans9 , thank you great information. 
Could you send us this pull request?
We have not iPhone X yet, so we can not test. (I truly want it)

**Comment by rmevans9** [comment]
Absolutely. I am playing with ways to fix the UI issue on the notes screen with no luck. I think it is largely due to how the screens are structured and how `native-base` works. 

You can reproduce the issue in the iPhone X simulator if that helps test it at all.

**Comment by kazup01** [comment]
I understand. I will try to fix it soon, thank you :)

**Comment by rmevans9** [comment]
PR up. The view is still broken so still needs some tweaking.
